### ISSUE TITLE
JSERB files lint the ruby but not the JS

### ISSUE THREAD

**Issue by ericpromislow** [Komodo/KomodoEdit#1038]
I'm working on a still top-secret udl extension that lets me use ERB to add compile-time
flags to JS code. It uses the ruby linter to find things like missing "<% end %>" stmts, but
doesn't show any JS errors or warnings. I turned on jslint, so I know there's some.

Turns out it's a partial-line fix. Will submit a PR.

**Comment by Naatan** [comment]
For what it's worth this may end up taking some additional work on our end, as we have updated our version of jshint to the latest and have removed jslint (in favour of jshint) for komodo 10.

Regardless, PR is more than welcome :)

Closing as there is no actual issue here. You should check out the forums ;)

**Comment by ericpromislow** [comment]
No, there's definitely an issue that has nothing to do with jslint (or jshint). The multi-language linter assumes it's in an HTML- or XML-based language. JSERB is JS-based, and it fails. Same thing would happen if you used ERB, EJS, or some other preprocessor to build your own CSS preprocessor. Please reopen.

**Comment by Naatan** [comment]
I see, thanks for clarifying.

**Comment by mitchell-as** [comment]
We'll await your PR then because it seems you know what needs to be done.

**Comment by Naatan** [comment]
@ericpromislow maybe you could share what you have with us, we might be able to finish it for you?
